### ISSUE TITLE
[Feature request]In the ios album sharing option, add a share button

### ISSUE THREAD

**Issue by bearcloney** [localsend/localsend#720]
### Is your feature request related to a problem? Please describe.

I hope to add the localsend share button to the options for sharing files in the iOS album.
希望在ios相册分享文件的选项中，加入localsend的分享按钮

This way you don’t have to jump from the picture to localsend and re-select the picture in order to share the picture.
这样不必为了分享图片而特地从图片跳转到localsend软件，再选择想分享的图片

Android can already share through this method, and I hope ios will also add it.
安卓已经可以通过这个办法分享了，希望ios也追加上

ios version 15.6, device ipadmini5

### Describe the solution you'd like

![IMG_0659](https://github.com/localsend/localsend/assets/51289114/20c87672-161f-4579-a384-d6ade78a7e7f)

**Comment by Tienisto** [comment]
There should be a LocalSend option. Maybe it is because you don't use the latest iOS version.

**Comment by bearcloney** [comment]
> 应该有一个 LocalSend 选项。可能是因为你没有使用最新的iOS版本。



> 应该有一个 LocalSend 选项。可能是因为你没有使用最新的 iOS 版本。

Yes, I just tried it with an ios16 device, and this option does appear. . . .
Can ios15 also support this function (my device is ios15.6)

是的 我刚刚用ios16的设备尝试了一下，确实出现了这个选项。。。。
请问能让ios15也支持这个功能吗（我的设备是ios15.6）
